.github/.workflow/test.yml

```diff
@@ -0,0 +1,13 @@
+name: Test and Deploy
+
+on:
+  push:
+    branches:
+      -main
```

Comment:
I know this was just an experiment...
You WILL need a space between - and main here